service/src/integrationTest/kotlin/fi/espoo/evaka/pis/service/MockVtjClientService.kt

```diff
@@ -0,0 +1,99 @@
+// SPDX-FileCopyrightText: 2017-2023 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.pis.service
+
+import fi.espoo.evaka.shared.dev.DevPerson
+import fi.espoo.evaka.vtjclient.service.vtjclient.IVtjClientService
+import fi.espoo.evaka.vtjclient.soap.VTJHenkiloVastaussanoma
+import java.util.concurrent.ConcurrentHashMap
+
+class MockVtjClientService : IVtjClientService {
+
+    companion object {
+        private var queryRequestResponse:
+            ConcurrentHashMap<
```

Comment:
Why not use normal HashMaps for `queryRequestResponse` and `queryCounts`? I don't think there's any situation where these will be accessed from multiple threads at the same time.

Reply:
This was requested in a PR review of the earlier implementation